### PR TITLE
Docs/epas/phase 3 reorg/fundamentals and planning

### PR DESCRIPTION
**Pull request by francoughlin**
## What Changed?

Edit the EPAS Fundamentals, Planning, Installing, and Upgrading sections

### CODE REVIEW

File: product_docs/docs/epas/15/fundamentals/sql_fundamentals/02_about_the_examples_used_in_this_guide.mdx [hunk 1]
```diff
@@ -11,15 +11,15 @@ redirects:
 
 <div id="about_the_examples_used_in_this_guide" class="registered_link"></div>
 
-The examples shown in the documentation use the PSQL program. The prompt that normally appears when using PSQL is omitted in these examples to provide extra clarity for the point being demonstrated.
+The examples shown in the documentation use the PSQL program. PSQL is a terminal based interface for PostgreSQL. The prompt that normally appears when using PSQL is omitted in these examples to provide extra clarity for the point being demonstrated.
```
Comment (nidhibhammar) [inline product_docs/docs/epas/15/fundamentals/sql_fundamentals/02_about_the_examples_used_in_this_guide.mdx:14]:
```suggestion
The examples shown in the documentation use the PSQL program. PSQL is a terminal-based command line interface for PostgreSQL. The prompt that normally appears when using PSQL is omitted in these examples to provide extra clarity for the point being demonstrated.
```

File: product_docs/docs/epas/15/fundamentals/sql_fundamentals/02_sql_tutorial/01_sample_database/01_sample_database_installation.mdx [hunk 2]
```diff
@@ -20,4 +20,4 @@ The tables and programs are created in the first schema of the search path in wh
 SHOW SEARCH_PATH;
 ```
 
-You can alter the search path using commands in PSQL.
+You can alter the search path using commands in PSQL (a terminal-based front-end to PostgreSQL).
```
Comment (nidhibhammar) [inline product_docs/docs/epas/15/fundamentals/sql_fundamentals/02_sql_tutorial/01_sample_database/01_sample_database_installation.mdx:23]: PSQL is used in both PostgreSQL and EPAS. I think it should be Postgres here to generalize
```suggestion
You can alter the search path using commands in PSQL (a terminal-based front-end to Postgres).
```

File: product_docs/docs/epas/15/fundamentals/sql_fundamentals/02_conventions.mdx [hunk 3]
```diff
@@ -7,13 +7,15 @@ redirects:
...
+
+The installation directory path of the PostgreSQL or EPAS products is referred to as *POSTGRES_INSTALL_HOME*.
 
 -   For PostgreSQL Linux installations, the default is `/opt/PostgreSQL/x.x` for version 10 and earlier. For later versions, use the PostgreSQL community packages.
--   For EDB Postgres Advanced Server Linux installations performed using the interactive installer for version 10 and earlier, the default is `/opt/edb/asx.x`.
--   For EDB Postgres Advanced Server Linux installations performed using an RPM package, the default is `/usr/edb/asxx`.
--   For EDB Postgres Advanced Server Windows installations, the default is `C:\Program Files\edb\asxx`. The product version number is represented by `x.x` or by `xx` for version 10 and later.
+-   For EPAS Linux installations performed using the interactive installer for version 10 and earlier, the default is `/opt/edb/asx.x`.
```
Comment (drothery-edb) [inline product_docs/docs/epas/15/fundamentals/sql_fundamentals/02_conventions.mdx:19]: I think we can remove this bullet. EPAS 10 is EOL'ed as are the interactive installers for Linux.
```suggestion
-   For EPAS Linux installations performed using the interactive installer for version 10 and earlier, the default is `/opt/edb/asx.x`.
```

File: product_docs/docs/epas/15/fundamentals/sql_fundamentals/03_advanced_concepts/06_multidimensional_analysis/03_grouping_sets_extension.mdx [hunk 4]
```diff
@@ -167,7 +171,7 @@ NEW YORK |            |           |         3
 (38 rows)
 ```
 
-The output is basically a concatenation of the result sets from `GROUP BY loc`, `GROUP BY ROLLUP (dname, job)`, and `GROUP BY CUBE (job, loc)`. These individual queries are shown by the following:
+The output is basically a concatenation of the result sets from `GROUP BY loc`, `GROUP BY ROLLUP (dname, job)`, and `GROUP BY CUBE (job, loc)`. These individual queries are shown below:
```
Comment (drothery-edb) [inline product_docs/docs/epas/15/fundamentals/sql_fundamentals/03_advanced_concepts/06_multidimensional_analysis/03_grouping_sets_extension.mdx:174]:
```suggestion
The output is basically a concatenation of the result sets from `GROUP BY loc`, `GROUP BY ROLLUP (dname, job)`, and `GROUP BY CUBE (job, loc)`. These individual queries are:
```

File: product_docs/docs/epas/15/fundamentals/sql_fundamentals/03_advanced_concepts/06_multidimensional_analysis/03_grouping_sets_extension.mdx [hunk 5]
```diff
@@ -177,7 +181,7 @@ GROUP BY loc
 ORDER BY 1;
 ```
 
-The following is the result set from the `GROUP BY loc` clause:
+The result set from the `GROUP BY loc` clause is shown below:
```
Comment (drothery-edb) [inline product_docs/docs/epas/15/fundamentals/sql_fundamentals/03_advanced_concepts/06_multidimensional_analysis/03_grouping_sets_extension.mdx:184]:
```suggestion
The result set from the `GROUP BY loc` clause is:
```

**Review by drothery-edb — APPROVED** [review]

File: product_docs/docs/epas/15/fundamentals/sql_fundamentals/02_sql_tutorial/01_sample_database/01_sample_database_installation.mdx (hunk 2 above)
Comment (francoughlin) [reply product_docs/docs/epas/15/fundamentals/sql_fundamentals/02_sql_tutorial/01_sample_database/01_sample_database_installation.mdx:23]: Got it, thanks, @nidhibhammar ! Fixing now, along with some other tweaks.

**Review by drothery-edb — DISMISSED** [review]

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**